amqp_influxdb/__init__.py

```diff
@@ -44,6 +45,24 @@ def __init__(self,
 
         connection_parameters = connection_parameters or {}
 
+        if isinstance(connection_parameters.get('credentials'), dict):
```

Comment:
They're not mandatory. If you don't supply them, pika defaults to guest/guest itself.